src/backend/config.ts

```diff
@@ -191,25 +191,30 @@ abstract class GlobalConfig {
     if (!isMac) {
       return wineSet
     }
-    const apps = readdirSync(`${userHome}/Applications/Wineskin`)
-    for (const app of apps) {
-      if (app.includes('.app')) {
-        const wineBin = `${userHome}/Applications/Wineskin/${app}/Contents/SharedSupport/wine/bin/wine64`
-        try {
-          const { stdout: out } = await execAsync(`'${wineBin}' --version`)
-          const version = out.split('\n')[0]
-          wineSet.add({
-            ...this.getWineExecs(wineBin),
-            lib: `${userHome}/Applications/Wineskin/${app}/Contents/SharedSupport/wine/lib`,
-            lib32: `${userHome}/Applications/Wineskin/${app}/Contents/SharedSupport/wine/lib`,
-            name: `Wineskin - ${version}`,
-            type: 'wine',
-            bin: wineBin
-          })
-        } catch (error) {
-          logError(`Error getting wine version for ${wineBin}`, {
-            prefix: LogPrefix.GlobalConfig
-          })
+    const wineSkinPath = `${userHome}/Applications/Wineskin`
+    if (existsSync(wineSkinPath)) {
```

Comment:
just a commend on this, maybe nit picky, but if you do something like:

```
if (existsSync(wineSkinPath)) {
  return wineSet
}

const apps = ....
```

then it will be easier to track changes on lines, if you nest the code in the if all lines will show up as if you changed them in this commit when it was actually just moved inside the if

not a big deal though (I do prefer early returns, but it's just my preference)